packages/mdc-checkbox/index.js

```diff
@@ -34,12 +34,12 @@ export class MDCCheckbox extends MDCComponent {
   }
 
   /**
-   * @return {InputElementState|undefined}
+   * @return {SelectionControlState|undefined}
```

Comment:
`{SelectionControlState|undefined}` -> `{?SelectionControlState}`

`querySelector()` returns `null`, not `undefined`.